packages/interfaces/src/events.ts

```diff
@@ -74,6 +74,10 @@ export class EventEmitter<EventMap extends { [s: string]: any }> extends EventTa
 
     return result
   }
+
+  safeDispatchEvent<Detail>(type: keyof EventMap, detail: CustomEventInit<Detail>): boolean {
```

Comment:
is `detail` the best name for this param?

Reply:
No idea, its already ref like that in multiple places

Reply:
`detail` is used in other places as it's the option key from the [CustomEvent](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent) constructor

Reply:
right it seems confusing since this is actually the CustomEvent options object which contains detail
https://github.com/libp2p/js-libp2p-interfaces/blob/2d7e06b882bed19c0ec208d05dd6404dc3d84de1/packages/interfaces/src/events.ts#L78-L79
https://github.com/libp2p/js-libp2p-interfaces/blob/2d7e06b882bed19c0ec208d05dd6404dc3d84de1/packages/interface-mocks/src/connection-manager.ts#L97-L99